### PR TITLE
LPS-133091 Adding new SAML test cases

### PR DESCRIPTION
**Pull request by ferenconodiD**
[LPS-133091](https://issues.liferay.com/browse/LPS-133091)

### CODE REVIEW

**Comment by ferenconodiD** [comment]
ci:test:sf

**Comment by ferenconodiD** [comment]
ci:test:security:nocompile

**Comment by ferenconodiD** [comment]
ci:sf:test

**Comment by ferenconodiD** [comment]
ci:test:sf

**Comment by ferenconodiD** [comment]
ci:test:sf

**Comment by ferenconodiD** [comment]
ci:test:sf

**Comment by gaborlovasdotcom** [comment]
ci:test:security:nocompile

**Comment by ferenconodiD** [comment]
ci:test:sf

**Comment by ferenconodiD** [comment]
ci:test:security:nocompile

**Comment by ferenconodiD** [comment]
ci:test:sf

**Comment by ferenconodiD** [comment]
ci:test:security:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase [hunk 1]
```diff
@@ -2461,6 +2504,591 @@ definition {
...
+		property app.server.bundles.size = "0";
+		property portal.upstream = "feri";
+		property test.name.skip.portal.instance = "SAML#ViewSPInitiatedSSOWithSAMLAttributeScreenNameUsedForUserMatchingMultipleIdP";
+
+		// For local testing
+		// BUG
+
+		var identityProviderEntityId1 = "samlidp1";
+		var identityProviderEntityId2 = "samlidp2";
+		var serviceProviderEntityId = "samlsp";
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase:2715]: Nice! 👍

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]
Review done, see my notes on the code.

**Comment by ferenconodiD** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/SAML.macro [hunk 2]
```diff
@@ -120,6 +120,25 @@ definition {
 		User.viewLoggedOutPG();
 	}
 
+	@description = "Add test user"
+	macro addTestUser {
+
+		// Add new test user
```
Comment (weller0246) [inline portal-web/test/functional/com/liferay/portalweb/macros/SAML.macro:126]: I think this comment is not necessary as the description says the same sentence

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase [hunk 3]
```diff
@@ -2461,6 +2504,596 @@ definition {
...
 	}
 
+	@description = "LPS-123218 TC-5: SP initiated login, using email address for user matching - Using virtual instances"
+	@priority = "4"
+	test ViewSPInitiatedSSOWithSAMLAttributeEmailAddressUsedForUserMatching {
+		property app.server.bundles.size = "0";
+		property portal.upstream = "true";
+		property test.name.skip.portal.instance = "SAML#ViewSPInitiatedSSOWithSAMLAttributeEmailAddressUsedForUserMatching";
+
+		// For local testing
```
Comment (weller0246) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase:2514]: Is this comment necessary in the repository?

**Review by weller0246 — CHANGES_REQUESTED** [review]

**Comment by ferenconodiD** [comment]
ci:test:sf

**Review by weller0246 — APPROVED** [review]

**Review by gaborlovasdotcom — APPROVED** [review]
LGTM!

**Comment by gaborlovasdotcom** [comment]
ci:test:saml:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**